containers/app/StorageSpaceStatus.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { CircularProgress, Dropdown, Icon, generateUID, useUser, usePopperAnchor } from 'react-components';
+import humanSize from 'proton-shared/lib/helpers/humanSize';
+
+const StorageSpaceStatus = () => {
+    const [{ MaxSpace, UsedSpace }] = useUser();
+    const [uid] = useState(generateUID('dropdown'));
+    const { anchorRef, isOpen, toggle, close } = usePopperAnchor();
+
+    const usedPercent = Math.round(UsedSpace / MaxSpace);
+    const maxSpaceFormatted = humanSize(MaxSpace);
+    const usedSpaceFormatted = humanSize(UsedSpace);
+
+    return (
+        <>
+            <CircularProgress
+                className="center"
+                progress={usedPercent}
+                aria-describedby={uid}
+                rootRef={anchorRef}
+                onClick={toggle}
+            >
+                <text
+                    x="50%"
+                    y="50%"
+                    fontFamily="Constantia"
+                    textAnchor="middle"
+                    alignmentBaseline="middle"
+                    stroke="#ffffff"
```

Comment:
judging from the image it's supposed to be `right-bottom`. The first part says on which side of target to place the dropdown, the second one says how it's aligned in relation to the target.

Glad my calculation works though